components/automate-ui/src/app/entities/api-tokens/api-token.effects.ts

```diff
@@ -1,6 +1,6 @@
 import { Injectable } from '@angular/core';
 import { HttpErrorResponse } from '@angular/common/http';
-import { Actions, Effect, ofType } from '@ngrx/effects';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
```

Comment:
the `@Effect` decorator has been deprecated in favor of a more "React hooks" style of composition.  
Deprecation notice: https://ngrx.io/api/effects/Effect
Usage notes for `createEffect`: https://ngrx.io/api/effects/createEffect#usage-notes